docs/manuals/auth/authorization.md

```diff
@@ -0,0 +1,254 @@
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+Authorization is the process of driving the application logic differently
+depending on the user using it: after a user is identified (see
```

Comment:
authenticated instead of identified?

Reply:
The problem is defining 'authentication' as a way to make users authenticated.
I tried to change the wording...

Let's see what the CS says, ok?

Reply:
I am good with identified.